### PR TITLE
adding new liquibase 

### PR DESCRIPTION
**Pull request by YanDeLeon**

### CODE REVIEW

File: tools/liquibase/generator.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	submatchall := re.FindAllString(name, -1)
+
+	for index, element := range submatchall {
+		if index == 0 {
+			underscoreName = strings.ToLower(element)
+			continue
+		}
+		underscoreName = underscoreName + "_" + strings.ToLower(element)
+	}
+	fullName := fecha + "-" + underscoreName + ".xml"
```
Comment (paganotoni) [inline tools/liquibase/generator.go:39]: @YanDeLeon this one should be done by an inflection library, see https://github.com/gobuffalo/flect/blob/master/underscore.go#L12

**Review by paganotoni — CHANGES_REQUESTED** [review]

**Comment by dannov91** [comment]
@paganotoni @YanDeLeon -- I love the shape it's taking!

File: plugins.go [hunk 2]
```diff
@@ -48,10 +50,12 @@ var Base = []plugins.Plugin{
 
 	// Generators
 	&ox.Generator{},
+	&liquibase.Generator{},
```
Comment (paganotoni) [inline plugins.go:53]: @YanDeLeon this one will be optional so no need to add it to the list of plugins.

**Review by paganotoni — CHANGES_REQUESTED** [review]

File: tools/liquibase/generator_test.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			t.Error("could not change to temp directory")
+		}
+
+		args := []string{"ox", "generate", "liquibase", "location/addDevices"}
+
+		g := Generator{}
+
+		err = g.Generate(context.Background(), root, args)
+
+		if err != nil {
```
Comment (paganotoni) [inline tools/liquibase/generator_test.go:41]: @YanDeLeon we need to check that the file gets generated and also check its content.

**Review by paganotoni — CHANGES_REQUESTED** [review]

**Review by paganotoni — APPROVED** [review]
Well done @YanDeLeon

**Pull request merged**